.github/ISSUE_TEMPLATE/flaky-ci-test-issue.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Flaky CI Test Issue
+about: Report a flaky test in the CI pipeline
+title: "[Flaky Test] <test-name>"
+labels: bug, flaky-test
+assignees: ''
+
+---
+
+**Description of the Flaky Test:**
+
+- **Test Name:** [Name of the test that is flaky]
+- **Test Location:** [File and line number or test suite]
+- **Frequency:** [How often does the test fail?]
+- **Recent CI Runs:** [Links to the recent CI runs where the test failed]
```

Comment:
Having client, client major version and piece of failure output could be handy

Reply:
As well